packages/angular/src/generators/mfe-host/schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/schema",
+  "$id": "NxMFEHost",
+  "cli": "nx",
+  "title": "Nx MFE Host App",
+  "description": "Create an Angular MFE Host App",
```

Comment:
nit: update the descriptions in this file to use full words so the docs look better